proto/hummock.proto

```diff
@@ -41,13 +41,27 @@ message InputLevel {
   repeated SstableInfo table_infos = 3;
 }
 
-message LevelDelta {
+message IntraLevelDelta {
   uint32 level_idx = 1;
   uint64 l0_sub_level_id = 2;
   repeated uint64 removed_table_ids = 3;
   repeated SstableInfo inserted_table_infos = 4;
 }
 
+message GroupConstruct {
+  CompactionConfig group_config = 1;
+}
+
+message GroupDestroy {}
```

Comment:
What about keep group id in GroupDestroy?

Reply:
> What about keep group id in GroupDestroy?

Currently not, because `GroupDestroy` is in `level_deltas`.
In the future, if we have `GroupSplit`/`GroupMerge` directly in `VersionDelta`(not `level_deltas`), we can then move `GroupDestroy` there and then keep group id in `GroupDestroy`